solidity/contracts/converter/BancorConverterRegistry.sol

```diff
@@ -186,13 +186,12 @@ contract BancorConverterRegistry is IBancorConverterRegistry, ContractRegistryCl
 
     /**
       * @dev adds an existing converter to the registry
-      * anyone can add an existing converter to the registry, as long as the converter is valid
-      * note that a liquidity pool converter can be added only if no converter with the same reserve-configuration is already registered
+      * can only be called by the owner
       * 
       * @param _converter converter
     */
-    function addConverter(IBancorConverter _converter) public {
-        require(isConverterValid(_converter) && !isSimilarLiquidityPoolRegistered(_converter));

```

Comment:
@yudilevi : There's a reason why `isSimilarLiquidityPoolRegistered` has been implemented separately from `isConverterValid`, so the fact that you removed it is troublesome. Can you please explain?

Reply:
we discussed it - feel free to resolve